functions/go/apply-setters/apply_setters_test.go

```diff
@@ -285,6 +285,59 @@ spec:
 `,
 			errMsg: `invalid setter pattern for array node: "${images}:${tag}"`,
 		},
+		{
+			name: "setter type handling",
+			config: `
+data:
+  foo: false
+  bar: 20
+  baz: 21.22
```

Comment:
Since we are putting these values into a ConfigMap, I'm pretty certain these values are not allowed. ConfigMaps only allow string values.